docs/user_guides/fs/storage_connector/usage.md

```diff
@@ -75,7 +103,7 @@ passing any SQL query to the `query` argument. This is mostly relevant for Googl
 === "Scala"
     ```scala 
     // read results from a SQL 
-    val df = connector.read("SQL", "csv" , new HashMap(),"")    
+    val df = connector.read("SELECT * FROM TABLE", "data_format" , new HashMap(),"")    
```

Comment:
do you need a format when reading a query?

Reply:
no, tested and removed now